internal/edge_cluster/cluster_profile_subresource.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 Broadcom. All Rights Reserved.
+// SPDX-License-Identifier: MPL-2.0
+
+package edge_cluster
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ClusterProfileSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the profile",
+			},
+			"bfd_allowed_hop": {
```

Comment:
Maybe it is worth checking out any constraints on the allowed values here (SDDCManager source code) to implement some kind of validation. True for the rest of the int attributes in this schema